### PR TITLE
Add win

### PR DESCRIPTION
**Pull request by oblute**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -59,6 +63,7 @@ requirements:
     - r-scales
     - r-tidyr >=0.6.1
     - r-xts
+    - r-bh
```
Comment (dbast) [inline recipe/meta.yaml:66]: r-bh is headers only and so only required during compilation in the host section and can be removed from run section.

**Comment by oblute** [comment]
@philip-khor @conda-forge/r this is ready for review and merge - i've built for windows now that r-stan has windows building.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,37 +27,49 @@ requirements:
...
-    - r-rstantools >=2.0.0
+    - r-rstantools ==2.0.0
     - r-scales
     - r-tidyr >=0.6.1
     - r-xts
+    - r-rcppparallel >=5.0.1
   run:
     - r-base
-    - {{ native }}gcc-libs         # [win]
+    - {{native}}gcc-libs         # [win]
```
Comment (philip-khor) [inline recipe/meta.yaml:58]:
```suggestion
    - {{ native }}gcc-libs         # [win]
```

**Review by philip-khor — APPROVED** [review]
Thanks! Just wondering if we need to deviate from the linter's recommendation here:

**Comment by oblute** [comment]
@philip-khor little change made

**Comment by philip-khor** [comment]
thank you!

**Pull request merged**